packages/cli/src/commands/proxy/consts.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * The reserved routes for Logto sign-in experience.
+ * Request to these routes will be proxied to the specified sign-in experience URL.
+ */
+export const LogtoSignInExperienceRoutes = [
```

Comment:
We can't, because they are going to call interaction apis from the custom ui, and if the custom ui and the interaction apis are not on the same origin, there will be a cors issue

Reply:
Ah, I think you are right. We can just proxy all the non-Logto requests to the custom ui.

Check removed.